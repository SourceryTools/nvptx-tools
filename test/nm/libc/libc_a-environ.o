// BEGIN PREAMBLE
.version 3.1
.target sm_35
.address_size 64
// END PREAMBLE
// BEGIN VAR DEF: initial_env
.global .align 8 .u64 initial_env[1];
// BEGIN GLOBAL VAR DEF: environ
.visible .global .align 8 .u64 environ[1] =
{generic(initial_env) };
