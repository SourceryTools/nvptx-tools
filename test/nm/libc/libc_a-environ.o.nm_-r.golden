0000000000000000 d initial_env
0000000000000000 D environ
